composer.json

```diff
@@ -14,11 +14,10 @@
         }
     ],
     "minimum-stability": "RC",
-
     "require": {
         "php": ">=5.6.0 <8.0",
         "codeception/lib-asserts": "^1.12.0",
-        "codeception/codeception": "*@dev"
```

Comment:
Please change this line back, it is important for builds of `codeception/codeception`.

Reply:
I did this change for all the repositories and they are already merged.
Should I revert back all the repositories?

Reply:
If you don't, the builds of `codeception/codeception` branches and pull requests will use the last version before your change.

Reply:
So codeception/codeception doesn't work already? because codeception/codeception has dependencies with: 
```
        "codeception/module-asserts": "*@dev",
        "codeception/module-cli": "*@dev",
        "codeception/module-db": "*@dev",
        "codeception/module-filesystem": "*@dev",
        "codeception/module-phpbrowser": "*@dev",
```

And module-db, module-cli, module-filesystem, module-phpbrowser use 
`"codeception/codeception": "^4.0"`

Reply:
It didn't break `codeception/codeception` because composer installs compatible versions.

Example from https://ci.appveyor.com/project/DavertMik/codeception/builds/32918812

```
  - Locking codeception/module-asserts (dev-master 79f13d0)
  - Locking codeception/module-cli (1.0.2)
  - Locking codeception/module-db (1.0.1)
  - Locking codeception/module-filesystem (1.0.2)
  - Locking codeception/module-phpbrowser (dev-lib-innerbrowser-branches 03182e0)
  - Locking codeception/phpunit-wrapper (7.8.0)
  - Locking codeception/specify (0.4.6)
  - Locking codeception/stub (3.0.0)
```

Older example: https://ci.appveyor.com/project/DavertMik/codeception/builds/32358191
```
 - Locking codeception/module-asserts (dev-master 79f13d0)
  - Locking codeception/module-cli (dev-master ca35844)
  - Locking codeception/module-db (dev-master 03c5337)
  - Locking codeception/module-filesystem (dev-master fe3c352)
  - Locking codeception/module-phpbrowser (dev-master 0b19576)
```

Reply:
Ok I don't get it, for me we don't need to have *@dev to be able to build codeception. And it was only useful when codeception 4 was not published yet. Now it's important to put the correct version to be sure that only user with codeception 4 could add the module.

Never mind, what should I do?
- revert only asserts
- revert asserts, db, cli, fileystem, phpbrowser
- revert the 28 packages

Reply:
It means that codeception/codeception builds and also docker images will keep using module-cli 1.0.2, module-db 1.0.1 and module-filesystem 1.0.2 until some new constraint causes version conflict, no matter what the latest version is.

Reply:
Yes and it's a good practice to only take the last release and not master branch.

Never mind, I revert the change but I think we should be consistent and do it everywhere not only in assert if it's a real issue for you.

- mongodb has a different constraint: "4.0.x-dev | ^4.0"
- assert: "*@dev"
- phpbrowser: "^4.0"